packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -1467,6 +1468,10 @@ export const useArbTokenBridge = (
     }
   }
 
+  function getRinkebyPivotBlock() {
+    return 13622530
```

Comment:
The follow-up log query might have `fromBlock` > `toBlock`
We can instead do something like
```suggestion
    const latest = await l1.signer.provider.getBlock('latest');
    const nitroStart = 13622530;
    return latest < nitroStart : await getL2GatewayGraphLatestBlockNumber(l1NetworkID) ? nitroStart;
```

Reply:
the block is before current block so we good (will just have some extra log query)